Update install instructions (#792)
DataDog/integrations-extras: cyral/README.md

### Installation

> If you are using Agent v6.8+ follow the instructions below to install the Redis's Sentinel check on your host. See our dedicated Agent guide for [installing community integrations][2] to install checks with the [Agent prior v6.8][3] or the [Docker Agent][4]:

Reviewer: ```suggestion
If you are using Agent v6.8+ follow the instructions below to install the Redis Sentinel check on your host. See the dedicated Agent guide for [installing community integrations][2] to install checks with the [Agent prior v6.8][3] or the [Docker Agent][4]:
```

> 1. [Download and launch the Datadog Agent][6].

Reviewer 2: ```suggestion
1. [Download and launch the Datadog Agent][9].
```
This is pointing to metadata csv

Check that the `url` in `cyral.yaml` is correct.

> Need help? Contact [Datadog support][9].

Reviewer 2: ```suggestion
Need help? Contact [Datadog support][8].
```
